frontend/src/views/WheresMyGeneV2/components/HeatMap/style.ts

```diff
@@ -161,16 +161,62 @@ export const YAxisWrapper = styled.div<YAxisWrapperProps>`
   overflow: hidden;
 `;
 
-export const StyledSkeleton = styled(Skeleton)`
-  margin: 0px 0px 4px 1px;
+interface LoadingContainerProps extends CommonThemeProps {
+  height: number;
+  width: number;
+}
+
+export const LoadingContainer = styled.div<LoadingContainerProps>`
+  position: absolute;
+  display: flex;
+  width: ${(props) => props.width * 20}px;
+  height: ${(props) => props.height * 20}px;
+  background-color: ${gray100};
 `;
 
-export const SkeletonContainer = styled.div`
+interface LoadingProps extends CommonThemeProps {
+  left: number;
+}
+
+export const LoadingWrapper = styled.div<LoadingProps>`
   display: flex;
   flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 600px;
+  left: ${(props) => ((props.left + 62) * 19) / 2}px;
```

Comment:
what's the intention behind this calculation? In case we need to adjust this in the future we'll know what this is trying to achieve and know what the goal is!